### PR TITLE
Update bsp for adding bt support

### PR DESCRIPTION
**Pull request by floion**
@agherzan @telphan @petrosagg

### CODE REVIEW

File: layers/meta-resin-artik/recipes-bsp/bcm4354-bluetooth/files/fwdown_adapt_to_resin.patch [hunk 1]
```diff
@@ -0,0 +1,38 @@
+Index: 1.0-r0/fwdown.sh
```
Comment (agherzan) [inline layers/meta-resin-artik/recipes-bsp/bcm4354-bluetooth/files/fwdown_adapt_to_resin.patch:7]: Add patch header.

File: layers/meta-resin-artik/recipes-bsp/bcm4354-bluetooth/bcm4354-bt.bbappend [hunk 2]
```diff
@@ -0,0 +1,15 @@
+FILESEXTRAPATHS_append := ":${THISDIR}/files"
+
+# add patch file that mainly moves the bluetooth mac address file into the resin-conf partition (needed for not clashing with resinhup)
```
Comment (agherzan) [inline layers/meta-resin-artik/recipes-bsp/bcm4354-bluetooth/bcm4354-bt.bbappend]: This description should be in the patch header and removed from here. Good content anyway.

**Comment by agherzan** [comment]
LGTM

**Comment by petrosagg** [comment]
lgtm

**Pull request merged**